From 4d2cfa9116b4651cf959e5a02feac0334590dbd9 Mon Sep 17 00:00:00 2001
From: Mel Gorman <mgorman@suse.de>
Date: Wed, 9 Mar 2011 19:23:55 +0000
Subject: [PATCH] collapse: mm: Report the low watermark correctly.
Patch-mainline: Not yet
References: bnc#678497

Report the correct low watermark plus the emergency pool offset
properly. Currently it is printing out an offset from the min watermark.

Signed-off-by: Mel Gorman <mgorman@suse.de>
Reviewed-by: NeilBrown <neilb@suse.de>
Signed-off-by: Suresh Jayaraman <sjayaraman@suse.de>
---
 mm/vmstat.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: linux-2.6.37-openSUSE-11.4/mm/vmstat.c
===================================================================
--- linux-2.6.37-openSUSE-11.4.orig/mm/vmstat.c
+++ linux-2.6.37-openSUSE-11.4/mm/vmstat.c
@@ -880,7 +880,7 @@ static void zoneinfo_show_print(struct s
 		   "\n        present  %lu",
 		   zone_page_state(zone, NR_FREE_PAGES),
 		   zone->pages_emerg + min_wmark_pages(zone),
-		   zone->pages_emerg + min_wmark_pages(zone),
+		   zone->pages_emerg + low_wmark_pages(zone),
 		   zone->pages_emerg + high_wmark_pages(zone),
 		   zone->pages_scanned,
 		   zone->spanned_pages,
